### PR TITLE
Rename python to ganga

### PR DESCRIPTION
**Pull request by mesmith75**
Test

### CODE REVIEW

**Comment by egede** [comment]
Does 

import ganga

at the python prompt still work after this change (and if this higher level dir is in the PYTHONPATH)?

File: setup.py [hunk 1]
```diff
@@ -79,8 +82,8 @@ def run(self):
       author_email='project-ganga-developers@cern.ch',
       license='GPL v2',
       scripts=['bin/ganga'],
-      package_dir={'': 'python'},
-      packages=find_packages('python'),
+      package_dir={'ganga': 'ganga'},
+      packages=allPackages,
```
Comment (alexanderrichards) [inline setup.py]: I believe that these two lines can now be simply:
```python
packages=find_packages()
```
i.e. package_dir is no longer needed

Comment (mesmith75) [reply setup.py]: yes

**Comment by mesmith75** [comment]
So I think this about works. It should replace #1188 

Edit: except it breaks `import ganga`

**Comment by mesmith75** [comment]
test this please

**Comment by mesmith75** [comment]
test this please

**Comment by mesmith75** [comment]
test this please

**Comment by mesmith75** [comment]
test this please

**Comment by mesmith75** [comment]
I *think* this is working fine now. The only change for python is `import ganga.ganga` rather than `import ganga`. Subsequently it is as normal, i.e. `print ganga.jobs` is the same.

**Review by egede — APPROVED** [review]

**Pull request merged**